src/lib/components/inventory/list_item_modal.ts

```diff
@@ -0,0 +1,474 @@
+                background: #66c0f4;
+            }
+
+            .auction-settings {
+                margin-top: 10px;
+            }
+
+            .duration-select {
+                width: 100%;
+                padding: 8px;
+                margin-top: 5px;
+                background: #2a475e;
+                border: 1px solid #000000;
+                color: #ffffff;
+            }
+
+            .description-input {
+                width: 100%;
+                padding: 8px;
+                margin-top: 5px;
+                background: #2a475e;
+                border: 1px solid #000000;
+                color: #ffffff;
+                resize: vertical;
+                min-height: 60px;
+            }
+
+            .checkbox-container {
+                margin-top: 10px;
+                display: flex;
+                align-items: center;
+                gap: 5px;
+            }
+
+            .success-content {
+                text-align: center;
+                padding: 20px 0;
+            }
+
+            .success-emoji {
+                font-size: 48px;
+                margin-bottom: 10px;
+            }
+
+            .success-title {
+                font-size: 18px;
+                color: #ffffff;
+                margin-bottom: 20px;
+            }
+
+            .success-links {
+                display: flex;
+                flex-direction: column;
+                gap: 10px;
+                margin-bottom: 20px;
+            }
+
+            .success-link {
+                padding: 10px;
+                background: #2a475e;
+                border: 1px solid #000000;
+                border-radius: 4px;
+                color: #ffffff;
+                text-decoration: none;
+                transition: background-color 0.2s;
+            }
+
+            .success-link:hover {
+                background: #3d6c8d;
+            }
+
+            .divider {
+                border-top: 1px solid #4b5f73;
+                margin: 20px 0;
+            }
+
+            .close-modal-button {
+                width: 100%;
+                padding: 10px;
+                background: #2a475e;
+                border: none;
+                border-radius: 4px;
+                color: #ffffff;
+                font-size: 16px;
+                cursor: pointer;
+                transition: background-color 0.2s;
+            }
+
+            .close-modal-button:hover {
+                background: #3d6c8d;
+            }
+
+            .tradable-warning {
+                background: rgba(255, 0, 0, 0.1);
+                border: 1px solid #ff4444;
+                border-radius: 4px;
+                padding: 15px;
+                margin-bottom: 20px;
+                color: #ff4444;
+                text-align: center;
+            }
+        `,
+    ];
+
+    async connectedCallback() {
+        super.connectedCallback();
+        // Only fetch price if the item is tradable
+        if (this.isTradable) {
```

Comment:
We'll need to add handling for whether they are currently logged into CSFloat.

You can check by making a call to `FetchPendingTrades` and see if the response is unauthorized or not.